vestapol/web_resources/base_resource.py

```diff
@@ -73,7 +71,7 @@ def requested_at_hive_path(self):
         return f'requested_at={self.requested_at.strftime("%Y-%m-%d %H:%M:%S")}'
 
     @staticmethod
-    def get_hive_path(data_path: List[Tuple[str, str]]):
+    def get_hive_path(data_path: List[Tuple[str, Any]]) -> str:
```

Comment:
Does `List[Tuple[str, str]]` not work because we're passing dates in the second tuple argument in some cases?

Reply:
yup. I could make it Union[str, datetime] if were sure those will be the only options